pandas/core/indexing.py

```diff
@@ -521,6 +521,10 @@ def loc(self) -> _LocIndexer:
         sidewinder mark i          10      20
                    mark ii          1       4
         viper      mark ii          7       1
+
+        For more details and explanations on advanced indexing, 
+        please visit the following link:
+        https://pandas.pydata.org/docs/user_guide/advanced.html#advanced-indexing-with-hierarchical-index
```

Comment:
Could you use a sphinx style reference like 

``` Please see the :ref:`user guide<advanced.advanced_hierarchical>` for more details and explanations of advanced indexing.```

Reply:
Sure, I went ahead and pushed the change to my repo